spacemesh/v1/admin_types.proto

```diff
@@ -131,6 +156,25 @@ message EventMalfeasance {
 
 message EventStreamRequest {}
 
+message EventRegisteredInPoet {
+  string url = 1;
+  bytes challenge = 2;
```

Comment:
I think this doesn't need to be part of the event. It's the hash of the header of the next ATX (not the hash/ID of the ATX), so not really useful to a user. However I think this event should include a `bytes smesher` field to know which identity was registered in PoET.

Reply:
That part was of requirements in the main [task](https://github.com/spacemeshos/go-spacemesh/issues/5565):
> registered in poet (URL, challenge, round)

If @poszu will agree - I can quickly remove `challenge`.
I'll `smasher`.

Reply:
Sure, the `challenge` is not critical. Unfortunately I don't recall why it was added. We can always add it later upon a community request. 

>However I think this event should include a bytes smesher field to know which identity was registered in PoET.

Probably all poet events should have it to tell which smesher given event is for, same as many other events. wdyt?